src/Symfony/Bridge/PhpUnit/bin/simple-phpunit

```diff
@@ -10,251 +10,4 @@
-    if ($components) {
-        array_shift($cmd);
-    }
-}
-
-$cmd[0] = sprintf('%s %s --colors=always', $PHP, escapeshellarg("$PHPUNIT_DIR/phpunit-$PHPUNIT_VERSION/phpunit"));
-$cmd = str_replace('%', '%%', implode(' ', $cmd)).' %1$s';
-
-if ('\\' === DIRECTORY_SEPARATOR) {
-    $cmd = 'cmd /v:on /d /c "('.$cmd.')%2$s"';
-} else {
-    $cmd .= '%2$s';
-}
-
-if ($components) {
-    $skippedTests = isset($_SERVER['SYMFONY_PHPUNIT_SKIPPED_TESTS']) ? $_SERVER['SYMFONY_PHPUNIT_SKIPPED_TESTS'] : false;
-    $runningProcs = array();
-
-    foreach ($components as $component) {
-        // Run phpunit tests in parallel
-
-        if ($skippedTests) {
-            putenv("SYMFONY_PHPUNIT_SKIPPED_TESTS=$component/$skippedTests");
-        }
-
-        $c = escapeshellarg($component);
-
-        if ($proc = proc_open(sprintf($cmd, $c, " > $c/phpunit.stdout 2> $c/phpunit.stderr"), array(), $pipes)) {
-            $runningProcs[$component] = $proc;
-        } else {
-            $exit = 1;
-            echo "\033[41mKO\033[0m $component\n\n";
-        }
-    }
-
-    while ($runningProcs) {
-        usleep(300000);
-        $terminatedProcs = array();
-        foreach ($runningProcs as $component => $proc) {
-            $procStatus = proc_get_status($proc);
-            if (!$procStatus['running']) {
-                $terminatedProcs[$component] = $procStatus['exitcode'];
-                unset($runningProcs[$component]);
-                proc_close($proc);
-            }
-        }
-
-        foreach ($terminatedProcs as $component => $procStatus) {
-            foreach (array('out', 'err') as $file) {
-                $file = "$component/phpunit.std$file";
-                readfile($file);
-                unlink($file);
-            }
-
-            // Fail on any individual component failures but ignore some error codes on Windows when APCu is enabled:
-            // STATUS_STACK_BUFFER_OVERRUN (-1073740791/0xC0000409)
-            // STATUS_ACCESS_VIOLATION (-1073741819/0xC0000005)
-            // STATUS_HEAP_CORRUPTION (-1073740940/0xC0000374)
-            if ($procStatus && ('\\' !== DIRECTORY_SEPARATOR || !extension_loaded('apcu') || !filter_var(ini_get('apc.enable_cli'), FILTER_VALIDATE_BOOLEAN) || !in_array($procStatus, array(-1073740791, -1073741819, -1073740940)))) {
-                $exit = $procStatus;
-                echo "\033[41mKO\033[0m $component\n\n";
-            } else {
-                echo "\033[32mOK\033[0m $component\n\n";
-            }
-        }
-    }
-} elseif (!isset($argv[1]) || 'install' !== $argv[1] || file_exists('install')) {
-    if (!class_exists('SymfonyBlacklistSimplePhpunit', false)) {
-        class SymfonyBlacklistSimplePhpunit {}
-    }
-    array_splice($argv, 1, 0, array('--colors=always'));
-    $_SERVER['argv'] = $argv;
-    $_SERVER['argc'] = ++$argc;
-    include "$PHPUNIT_DIR/phpunit-$PHPUNIT_VERSION/phpunit";
-}
-
-exit($exit);
+require dirname(__DIR__).DIRECTORY_SEPARATOR.'simple-phpunit.php';
```

Comment:
I'd suggest putting the new file in the `bin/` directory

Reply:
Done